src/azure-cli/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -1257,6 +1258,15 @@ def build_vmss_resource(cmd, name, computer_name_prefix, location, tags, overpro
         if spot_restore_timeout:
             vmss_properties['spotRestorePolicy']['restoreTimeout'] = spot_restore_timeout
 
+    if (regular_priority_count is not None or regular_priority_percentage is not None) and \
+            cmd.supported_api_version(min_api='2022-08-01', operation_group='virtual_machine_scale_sets'):
```

Comment:
>  cmd.supported_api_version(min_api='2022-08-01', operation_group='virtual_machine_scale_sets'):

We have added the setting of `min_api` to the definition of parameters. Why do we need to add the `cmd.supported_api_version` to the logic?

Reply:
Deleted